documentation/API-design-guidelines.md

```diff
@@ -1030,6 +1030,7 @@ This part describes the list of possible messages returned by the API. It also i
 This part captures a detailed description of all the data structures used in the API specification. For each of these data, the specification must contain:
 - Name of the data object, used to reference it in other sections.
 - Data type (String, Integer, Object…).
+- If the format of a string is datetime following sentence must be present in the description: `It must follow RFC 3339 and must have time zone. Recommended format is yyyy-MM-dd'T'HH:mm:ss.SSSZ (i.e. which allows 2023-07-03T14:27:08.312+02:00 or 2023-07-03T12:27:08.312Z)`
```

Comment:
In other parts of design guidelines some references to RFCs are enriched with hyperlink to the source on IETF webpage, so here it would look like:
```suggestion
- If the format of a string is datetime following sentence must be present in the description: `It must follow [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) and must have time zone. Recommended format is yyyy-MM-dd'T'HH:mm:ss.SSSZ (i.e. which allows 2023-07-03T14:27:08.312+02:00 or 2023-07-03T12:27:08.312Z)`
```